package.json

```diff
@@ -7,7 +7,7 @@
   "publisher": "hashicorp",
   "license": "MPL-2.0",
   "engines": {
-    "vscode": "^1.46.0"
+    "vscode": "^1.50.1"
```

Comment:
Is there a particular feature/change between these versions you were hoping would address the test failures?

Reply:
I've been keeping this set to a six-month window so that we aren't trying to maintain compatibility way outside of what people are likely to be using. It doesn't affect the test issues.